wasabi/research-club/anonymous-credentials.md

```diff
@@ -1,695 +1,327 @@
 
-Speaker 0: 00:07:28
+Jonas  7:Unknown speaker8  
+If I may add to that, I would say that Brands credentials are also on unlinkable if you can talk to the server, right? Because that's what you also usually do in e-cash, you basically show the serial number of your old token, you create a new token, you show that the tokens are the same in zero knowledge, and then you get a signature on the new token. And that way, you get unlinkable token. I think the advantage with Anonymous credentials slight is that you can do that without having to talk to the server.
 
-If I may add to that, I would say that brands credentials are also unlinkable.
-If you can talk to the server, right?
-Because that's what you also usually do in e-cash.
-You basically show the serial number of your old token.
-You create a new token.
-You show that the tokens are the same in zero knowledge, and then you get a signature on the new token and that way you get an unlinkable token.
-I think the advantage with anonymous credentials is that you can do that without having to talk to the server.
+## Mercurial signatures (Crites, Lysyanskaya, Unknown speaker0Unknown speaker0). / Delegatable Anonymous Credentials.
 
-## Mercurial signatures (Crites, Lysyanskaya, 2020). / Delegatable Anonymous Credentials.
-
-Speaker 1: 00:08:04
-
-Yes, it's called I think Ray Schwentz or that's what usually comes up.
-Anyway, so a bit more there because the same authors were working on this for probably a decade after this paper and they actually came up with something called Mercurial Signatures where they don't need so much crypto as much as in this paper, but in a very straightforward way they could achieve the exact same thing with these mercurial signatures.
-And even more, they were working on something called delegatable anonymous credentials.
-And the trick is that you could actually delegate the ownership of the credential to someone else.
-So in our case, that would be, I think, giving money to someone and I wouldn't be able to redeem it, but you would, which is actually pretty nice.
-Oh my God, I wasn't talking.
-
-Speaker 0: 00:09:26
+Speaker 1  8:03  
+Yes, it's called I think, reissuance or that's what usually comes up. Anyway. So a bit more there. Because the same outdoors were working on this for probably a decade of this paper and, and they actually came up with something called mercury, our signatures where they don't need so much crypto as much as in this paper. But what in a very straightforward way they could achieve the exact same thing with this mercury as signatures. And even more. They were working on some things called delegated anonimous credentials, and they're the three keys that you could actually delegate the ownership of the credential to someone else. So in our case, that would be I think, giving money to someone, and I wouldn't be able to redeem it, but you would, which is actually pretty nice. Oh, my God, I wasn't talking?
```

Comment:
You correctly catch the wrongly transcribed "Ray Schwentz" and fixed it to  "reissuance" but for the rest of this text you introduced a bunch of mistakes. For example:
- "authors" to "outdoors"
- "Mercurial Signatures" to "mercury, our"
- "and the trick is that" to "and the're the three keys that"

Reply:
How do I make changes?

Reply:
You can do the changes through the website editor in the same way that you initially reviewed and submitted the transcript.